dataflow-website/batch-developer-guides/batch/batchsamples/billrun/pom.xml

```diff
@@ -45,6 +45,12 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
+			<exclusions>
```

Comment:
I don't really see no point to exclude these unless we want to make sure there's no junit4 tests?

Reply:
Didn't realise until just now that initializr does this same so now I understand why you did it.